source/iterators.tex

```diff
@@ -1543,7 +1543,7 @@
 \pnum
 The \libconcept{sized_sentinel_for} concept specifies
 requirements on an \libconcept{input_or_output_iterator} and
-a corresponding \libconcept{sentinel_for}
+a corresponding sentinel
```

Comment:
It would be good to keep the "is-a sentinel_for" here. Maybe use a partial-concept-id: "a corresponding `sentinel_for<I>`" after introducing "I" for the input_or_output_iterator.

Reply:
Sounds like the best solution, if I'm able to introduce `I` without too much noise.